### PR TITLE
Add Roadmap to README

### PR DESCRIPTION
**Pull request by mark-burnett**
This is still perhaps a bit thin, but adds a minimal Roadmap with room for improvement.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,25 @@
-# Overview
+# Promenade
 
-Promenade is tool for deploying self-hosted, highly resilient Kubernetes clusters.
+Promenade is a tool for bootstrapping a resilient Kubernetes cluster and
+managing cluster life-cycle.
```
Comment (intlabs) [inline README.md]: s/cluster/its I think - as otherwise its not clear which cluster promenade is managing.

**Review by intlabs — CHANGES_REQUESTED** [review]
looks great - as discussed I think moving the getting started/dev doc to another file would make this a lot clearer.

File: README.md [hunk 2]
```diff
@@ -1,85 +1,31 @@
...
-export http_proxy=...
-export no_proxy=...
-docker build --build-arg http_proxy=$http_proxy --build-arg https_proxy=$http_proxy --build-arg no_proxy=$no_proxy  -t quay.io/attcomdev/promenade:experimental .
-```
-
-## Development Cleanup
-
-If you are testing/developing on hosts that cannot be easily destroyed, you may
-find the `cleanup.sh` script useful.
+Bugs are tracked in
```
Comment (intlabs) [inline README.md:28]: are bugs tracked here or in jira?

**Review by intlabs — COMMENTED** [review]
One minor thing - otherwise looks good.

File: Docs/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
+# Getting Started
```
Comment (gardlt) [inline Docs/getting-started.md]: nit: Docs - > docs

File: README.md (hunk 2 above)
Comment (intlabs) [reply README.md:28]: After reading - and comprehending - I'm good. :)

**Review by intlabs — APPROVED** [review]
LGTM :)

File: Docs/getting-started.md (hunk 3 above)
Comment (mark-burnett) [reply Docs/getting-started.md]: Done.

**Review by gardlt — APPROVED** [review]

**Comment by eanylin** [comment]
Looks good

**Pull request merged**